test/extended/util/image/image.go

```diff
@@ -20,6 +20,10 @@ func init() {
 		// used by oc mirror test, should be moved to publish to quay
 		"docker.io/library/registry:2.7.1": -1,
 
+		// used by build s2i e2e's to verify that builder with USER root are not allowed, should be moved to quay
```

Comment:
why should it be moved to quay?  it's not on docker.io so i would think that's good enough.

Reply:
forgot to clean up that comment as part of our ongoing education here 

just pushed an update that removed that quay related phrase